### PR TITLE
fix: add signature suite when ldpSuites is empty

### PR DESCRIPTION
**Pull request by fqutishat**
Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/doc/signature/verifier/public_key_verifier.go [hunk 1]
```diff
@@ -158,12 +158,16 @@ func NewEd25519SignatureVerifier() *Ed25519SignatureVerifier {
 
 // Verify verifies the signature.
 func (sv Ed25519SignatureVerifier) Verify(pubKey *PublicKey, msg, signature []byte) error {
+	value := pubKey.Value
+	if pubKey.JWK != nil {
```
Comment (troyronda) [inline pkg/doc/signature/verifier/public_key_verifier.go:162]: this should be done in constructor

File: pkg/doc/signature/verifier/public_key_verifier.go [hunk 2]
```diff
@@ -158,12 +158,16 @@ func NewEd25519SignatureVerifier() *Ed25519SignatureVerifier {
...
+		value = pubKey.JWK.Public().Key.(ed25519.PublicKey)
```
Comment (Baha-sk) [inline pkg/doc/signature/verifier/public_key_verifier.go:163]: need to add `ok` check

Comment (fqutishat) [reply pkg/doc/signature/verifier/public_key_verifier.go]: done

File: pkg/doc/signature/verifier/public_key_verifier.go [hunk 3]
```diff
@@ -158,12 +158,22 @@ func NewEd25519SignatureVerifier() *Ed25519SignatureVerifier {
...
 // Verify verifies the signature.
 func (sv Ed25519SignatureVerifier) Verify(pubKey *PublicKey, msg, signature []byte) error {
+	value := pubKey.Value
+
+	if pubKey.JWK != nil {
+		var ok bool
+		value, ok = pubKey.JWK.Public().Key.(ed25519.PublicKey)
+
+		if !ok {
+			return fmt.Errorf("public key not ed25519.PublicKe")
```
Comment (Baha-sk) [inline pkg/doc/signature/verifier/public_key_verifier.go:168]: `PublicKe` typo

Comment (fqutishat) [reply pkg/doc/signature/verifier/public_key_verifier.go]: done

**Pull request merged**